ogarniacz_mp3.py

```diff
@@ -38,6 +40,21 @@ def get_format_preference_index(ext):
     except ValueError:
         return len(format_preference)
 
+
+def convert_to_mp3(processing_dir_path):
+    for dir_path, sub_dirs, files in os.walk(processing_dir_path):
+        for file_name in files:
+            file_path = os.path.join(dir_path, file_name)
+            if classes.MusicFile.is_music_file(file_path):
+                name, ext = split_file_name(file_name)
+                if ext != "mp3":
+                    input_path = file_path
+                    output_path = os.path.join(dir_path, name + ".mp3")
+                    subprocess.call(["ffmpeg", "-i", input_path, "-vn", "-ar", 44100, "-ac", 2, "-b:a", 142000, output_path],
+                    stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE)
```

Comment:
Zamiast `asyncio.subprocess.PIPE` użyj `subprocess.PIPE`. Podejrzewam, że to rozwiąże twój aktualny problem.

Reply:
Nie rozwiązało - dostaję ten sam błąd.

Reply:
Ok - mój błąd. Trzeba zamienić wszystkie liczby w tej komendzie na stringi. Więc będzie

```python
subprocess.call(["ffmpeg", "-i", input_path, "-vn", "-ar", "44100", "-ac", "2", "-b:a", "142000", output_path],
                        stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Reply:
Teraz jest dobrze.